lib/plugins/noop.js

```diff
@@ -0,0 +1,27 @@
+var noop = function(worker, func, queue, job, args, options){
+  var self = this;
+  self.name = 'noop';
+  self.worker = worker;
+  self.queue = queue;
+  self.func = func;
+  self.job = job;
+  self.args = args;
+  self.options = options;
+
+  if(self.worker.queueObject){
+    self.queueObject = self.worker.queueObject;
+  }else{
+    self.queueObject = self.worker;
+  }
+};
+
+noop.prototype.after_perform = function(callback){
+  var self = this;
+  if(self.worker.error){
+    delete self.worker.error;
```

Comment:
This is a great use for `options`. 

```
if(typeof self.options.logger === 'function'){
  self.options.logger(error);
}else{
  console.log(error);
}
```

Here's an example of passing options to a plugin via a job https://github.com/taskrabbit/node-resque/blob/master/examples/retry.js#L27-L45